### DISCUSSION TITLE
修改Form值的时候，视图不会响应式更新， 代码改编自Formily Element Plus，FormDialog组件

### DISCUSSION THREAD

**Discussion by heianxiyi** [alibaba/formily#3915] [Q&A 2.x]
# 组件代码：

## 渲染Form
<img width="650" alt="image" src="https://github.com/alibaba/formily/assets/22736955/b07840d8-6d2e-4485-b9f4-d24fded5b402">
## 渲染FormFooter
<img width="571" alt="image" src="https://github.com/alibaba/formily/assets/22736955/15657cb7-3ba5-4f72-b2e7-b6deae777dc2">

# 之前：

<img width="1205" alt="image" src="https://github.com/alibaba/formily/assets/22736955/4add6f53-7f72-466a-a8e0-764bf1c4fd1d">

## 这里修改Form值的时候，视图不会响应式更新

# 之后：

<img width="1129" alt="image" src="https://github.com/alibaba/formily/assets/22736955/74aa4874-852e-4599-b108-31f34a0e8256">

## 这里修改Form值的时候，视图会响应式更新

# FormConsumer 源码

<img width="1040" alt="image" src="https://github.com/alibaba/formily/assets/22736955/1e4f297a-2135-44f2-ac40-e4f449c7dc90">

**Comment by nigiwen** [comment]
为啥我的footer插槽直接用不了了
